File: DownloadAPI/GetImageForDownload.js
let data = null;
const xhr = new XMLHttpRequest();

const lat = 55.862261;
const lng = -4.260959;
const zoom = 16;
const imageId = "ADD IMAGE ID HERE";
const token = "ADD YOUR TOKEN HERE";

const url = "https://download-api.hibirdi.com/download/lat/" + lat +
            "/lng/" + lng +
            "/zoom/" + zoom +
            "/id/" + imageId +
            "?token=" + token;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === 4) {
    console.log(this.status);
  }
});

xhr.open("GET", url);
xhr.send(data);

File: DownloadAPI/GetImageForDownload.py
import requests

lat = 55.862261
lng = -4.260959
zoom = 16
imageId = "ADD IMAGE ID HERE"
token = "ADD YOUR TOKEN HERE"

url = ("https://download-api.hibirdi.com/download/lat/{}/lng/{}/zoom/{}/id/{}?token={}"
       .format(lat, lng, zoom, imageId, token))

response = requests.request("GET", url)

print(response.text)
